mon-pix/app/components/module/element/_short-video.scss

```
@@ -0,0 +1,8 @@
+.element-short-video {
+
+  &__video {
+    aspect-ratio: auto 16 / 9;
+    width: 100%;
+  border-radius: var(--pix-spacing-2x);
```

Bizarre que le linter n'ait pas vu ça

```suggestion
    border-radius: var(--pix-spacing-2x);
```

---

J'ai corrigé à la main mais oui bizarre 🤔